### PR TITLE
tools: update descriptions for query-status of DM

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? 
`query-status` command for dmctl has been changed. The docs should be updated.

### What is the related PR or file link(s)? 
https://github.com/pingcap/dm/pull/334

### Which version does your change affect? 
update dev now, and after get two LGTM I will update v2.1 and v3.0

### CODE REVIEW

File: dev/reference/tools/data-migration/query-status.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ category: reference
 
 # DM 查询状态
 
-本文介绍 DM（Data Migration）`query-status` 命令的查询结果以及子任务状态。
+本文介绍 DM（Data Migration）`query-status` 命令的查询结果，任务状态与子任务状态。
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
本文介绍 DM（Data Migration）`query-status` 命令的查询结果、任务状态与子任务状态。
```

File: dev/reference/tools/data-migration/query-status.md [hunk 2]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+| 任一 Sync 类型子任务处于 “Running” 状态但其 Relay 插件已停止运行（处于 Error/Paused/Stopped 状态） | Error - Relay status is Error/Paused/Stopped |
+| 任一子任务处于 “Paused” 状态且返回结果没有错误信息 | Paused |
+| 所有子任务处于 “New” 状态 | New |
+| 所有子任务处于 “Finished” 状态 | Finished |
+| 所有子任务处于 “Stopped” 状态 | Stopped |
+| 其他情况 | Running |
+
+## 详情查询结果
+
+```bash
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]: 在这行前加一个 `{{< copyable "" >}}` 以及空行

File: dev/reference/tools/data-migration/query-status.md [hunk 3]
```diff
@@ -15,6 +15,65 @@ category: reference
...
 ```
 
+```
+{
+    "result": true, # 查询是否成功。
+    "msg": "",      # 查询失败原因描述。
+    "tasks": [      # 迁移 task 列表
+        {
+            "taskName": "test-1",           # 任务名称
+            "taskStatus": "Running",        # 任务运行状态，包括 “New”，“Running”，“Paused”，“Stopped”，“Finished”以及“Error”。
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
            "taskStatus": "Running",        # 任务运行状态，包括 “New”，“Running”，“Paused”，“Stopped”，“Finished” 以及 “Error”。
```

File: dev/reference/tools/data-migration/query-status.md [hunk 4]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+            "workers": [                    # 该任务所使用的 DM-workers 列表
+                "127.0.0.1:8262"
+            ]
+        },
+        {
+            "taskName": "test-2",
+            "taskStatus": "Error - Some error occurred in subtask", # 该任务的子任务存在运行错误并终止的现象
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]: `暂停` 是不是比 `终止` 更合适？

File: dev/reference/tools/data-migration/query-status.md [hunk 5]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+            "workers": [
+                "127.0.0.1:8262",
+                "127.0.0.1:8263"
+            ]
+        },
+        {
+            "taskName": "test-3",
+            "taskStatus": "Error - Relay status is Error",  # 该任务的某个 Sync 类型的子任务在同步时 Relay 插件出错
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
            "taskStatus": "Error - Relay status is Error",  # 该任务的某个处于 Sync 阶段的子任务对应的 Relay 处理单元出错
```

File: dev/reference/tools/data-migration/query-status.md [hunk 6]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+                "127.0.0.1:8264"
+            ]
+        }
+    ]
+}
+```
+
+关于 tasks 下的 taskStatus 状态的详细定义，请参阅[任务状态](#任务状态)。
+
+较为推荐的 query-status 使用方法是：
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
较为推荐的 `query-status` 使用方法是：
```

File: dev/reference/tools/data-migration/query-status.md [hunk 7]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+
+1. 首先使用 query-status 查看各个 task 的运行状态是否正常。
+2. 如果发现其中某一 task 状态有问题，通过 query-status `出错任务的 taskName` 来得到更详细的错误信息。
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
2. 如果发现其中某一 task 状态有问题，通过 `query-status <出错任务的 taskName>` 来得到更详细的错误信息。
```

File: dev/reference/tools/data-migration/query-status.md [hunk 8]
```diff
@@ -15,6 +15,65 @@ category: reference
...
+
+## 任务状态
+
+DM 的迁移任务状态取决于其分配到 DM-worker 上的[子任务状态](#子任务状态)，定义见下表：
+
+| 任务对应的所有子任务的状态 | 任务状态 |
+| :---: | :---: |
+| 任一子任务处于 “Paused” 状态且返回结果有错误信息 | Error - Some error occurred in subtask |
+| 任一 Sync 类型子任务处于 “Running” 状态但其 Relay 插件已停止运行（处于 Error/Paused/Stopped 状态） | Error - Relay status is Error/Paused/Stopped |
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
| 任一处于 Sync 阶段的子任务处于 “Running” 状态但其 Relay 处理单元未运行（处于 Error/Paused/Stopped 状态） | Error - Relay status is Error/Paused/Stopped |
```

File: dev/reference/tools/data-migration/query-status.md (hunk 4 above)
Comment (lichunzhu) [reply dev/reference/tools/data-migration/query-status.md]: 是

**Comment by csuzhangxc** [comment]
@yikeke PTAL

File: dev/how-to/get-started/data-migration.md [hunk 9]
```diff
@@ -291,6 +291,7 @@ port = 3307
 
 - 如果从 MySQL Server、Percona Server、Percona XtraDB Cluster、Amazon Aurora 或 RDS 迁移数据，则 `flavor` 配置项应设为 "mysql"（默认值，支持 5.5 < MySQL 版本 < 8.0）。
 - 如果从 MariaDB Server 或 MariaDB (Galera) Cluster 迁移数据，则设置 `flavor = "mariadb"`（仅支持 10.1.2 以上 MariaDB 版本）。
+- 在 v1.0.2 版本后 `flavor`，`server-id` 项均会由 DM 自动生成，一般情况下不需要手动配置。
```
Comment (yikeke) [inline dev/how-to/get-started/data-migration.md]:
```suggestion
- 从 DM 1.0.2 版本开始，`flavor`、`server-id` 项均会由 DM 自动生成，一般情况下不需要手动配置。
```

File: dev/how-to/migrate/from-aurora.md [hunk 10]
```diff
@@ -126,7 +126,7 @@ black-white-list:
 # Mydumper 全局配置，各实例通过配置项名引用。
 mydumpers:
   global:
-    extra-args: "-B test_db -T test_table"  # 只导出 `test_db` 库中的 `test_table` 表，可设置 Mydumper 的任何参数。
+    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，在 v1.0.2 版本中 DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```
Comment (yikeke) [inline dev/how-to/migrate/from-aurora.md]:
```suggestion
    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，从 DM 1.0.2 版本开始，DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```

File: dev/reference/tools/data-migration/deploy.md [hunk 11]
```diff
@@ -98,7 +98,7 @@ black-white-list:
 mydumpers:
   global:
     mydumper-path: "./bin/mydumper"   # Mydumper 二进制文件的路径。
-    extra-args: "-B test_db -T test_table"  # 只导出 `test_db` 库中的 `test_table` 表，可设置 Mydumper 的任何参数。
+    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，在 v1.0.2 版本中 DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```
Comment (yikeke) [inline dev/reference/tools/data-migration/deploy.md]:
```suggestion
    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，从 DM 1.0.2 版本开始，DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```

File: dev/reference/tools/data-migration/from-aurora.md [hunk 12]
```diff
@@ -126,7 +126,7 @@ black-white-list:
 # Mydumper 全局配置，各实例通过配置项名引用。
 mydumpers:
   global:
-    extra-args: "-B test_db -T test_table"  # 只导出 `test_db` 库中的 `test_table` 表，可设置 Mydumper 的任何参数。
+    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，在 v1.0.2 版本中 DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```
Comment (yikeke) [inline dev/reference/tools/data-migration/from-aurora.md]:
```suggestion
    extra-args: "-B test_db -T test_table"  # mydumper 的其他参数，从 DM 1.0.2 版本开始，DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置。
```

File: dev/reference/tools/data-migration/query-status.md [hunk 13]
```diff
@@ -15,6 +15,67 @@ category: reference
...
+                "127.0.0.1:8264"
+            ]
+        }
+    ]
+}
+```
+
+关于 tasks 下的 taskStatus 状态的详细定义，请参阅[任务状态](#任务状态)。
+
+较为推荐的 `query-status` 使用方法是：
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
推荐的 `query-status` 使用方法是：
```

File: dev/reference/tools/data-migration/query-status.md [hunk 14]
```diff
@@ -15,6 +15,67 @@ category: reference
...
+
+1. 首先使用 query-status 查看各个 task 的运行状态是否正常。
+2. 如果发现其中某一 task 状态有问题，通过 `query-status <出错任务的 taskName>` 来得到更详细的错误信息。
+
+## 任务状态
+
+DM 的迁移任务状态取决于其分配到 DM-worker 上的[子任务状态](#子任务状态)，定义见下表：
+
+| 任务对应的所有子任务的状态 | 任务状态 |
+| :---: | :---: |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
| :--- | :--- |
```
列表统一左对齐

**Comment by csuzhangxc** [comment]
@yikeke PTAL again

**Pull request merged**